iroh/src/net_report.rs

```diff
@@ -475,8 +475,19 @@ impl Client {
 
         let mut reports = Vec::new();
 
+        // We set _did_start to true if at least on report was started for each category..
+        let ipv4_started = !v4_buf.is_empty();
+        let ipv6_started = !v6_buf.is_empty();
+        // If we did not start any report for either category, _did_finish is set to false right away
+        // (it "completed" in the sense that nothing will ever run). If we did start at least one report,
```

Comment:
Nit: We could invert the boolean logic and call this `ipv4_pending` and `ipv6_pending` instead.

Reply:
oh yeah, that's simpler to follow indeed. Feel free to push or I can do tonight (or as a followup if this gets merged before)